src/InteractiveWindow/VisualStudio/InteractiveWindow.vsct

```diff
@@ -285,7 +296,9 @@
     <KeyBinding guid="guidInteractiveWindowCmdSet" id="cmdidSearchHistoryPrevious" key1="VK_UP" mod1="Control Alt" editor="guidInteractiveWindow" />
     <KeyBinding guid="guidInteractiveWindowCmdSet" id="cmdidCopyCode" key1="C" mod1="Control Shift" editor="guidInteractiveWindow" />
 
-    <KeyBinding guid="guidVSStd11" id="cmdidExecuteSelectionInInteractive" editor="guidCSharpEditorFactory" mod1="Control" key1="E" mod2="Control" key2="E" />
+    <!-- TODO: Revert to the code below pending https://github.com/dotnet/roslyn/issues/8927 .
+         <KeyBinding guid="guidVSStd11" id="cmdidExecuteSelectionInInteractive" editor="guidCSharpEditorFactory" mod1="Control" key1="E" mod2="Control" key2="E" />-->
+    <KeyBinding guid="guidInteractiveWindowCmdSet" id="cmdidExecuteInInteractiveWindow" editor="guidCSharpEditorFactory" mod1="Control" key1="E" mod2="Control" key2="E" />
```

Comment:
Does this implicitly add Ctrl-E,E as well?  I think it's conventional to have both.